### ISSUE TITLE
Unexpected token { inside a document in the src folder

### ISSUE THREAD

**Issue by nehakrishnakumar** [nehakrishnakumar/ban-me-bot#1]
At the moment, I do not know how to resolve this error. It should not have to do with my code at all, since it seems to be with the Discord library itself. I think it is possible that I inputted the token in improperly. I will resolve this within the next week.

**Comment by nehakrishnakumar** [comment]
I switched from hard-coding the token in index.js and instead moved it toward config.json. However, the error is still coming. I need to figure out what the requires() method entails to see if I am using this correctly. Chances are, that specific line of code is the one that Discord is taking issues toward. I am expecting another email from Discord saying that they found my token being used online. Hopefully I will not receive this email because I added a line to .gitignore to ignore the config.json file. Either way, I do not see it on here, which is a good sign. I am still stumbling through the idea of tokens and API keys since this is only the second time I've used an API before.
